plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/ParallelFileSystemAccess.xtend

```diff
@@ -10,17 +10,18 @@ import org.eclipse.xtext.generator.IFileSystemAccessExtension2
 import org.eclipse.xtext.generator.IFileSystemAccessExtension3
 import org.eclipse.xtext.resource.IResourceDescription.Delta
 import org.eclipse.xtext.util.RuntimeIOException
+import org.eclipse.xtext.generator.IFileSystemAccessExtension4
 
 /**
  * @author Anton Kosyakov
  * @since 2.7
  */
-class ParallelFileSystemAccess implements IFileSystemAccess, IFileSystemAccessExtension, IFileSystemAccessExtension2, IFileSystemAccessExtension3 {
+class ParallelFileSystemAccess implements IFileSystemAccess, IFileSystemAccessExtension, IFileSystemAccessExtension2, IFileSystemAccessExtension3, IFileSystemAccessExtension4 {
 
 	Delta delta
-	
```

Comment:
Please avoid reformatting entire files in the future as it makes reviews more time consuming than necessary.